### PR TITLE
Set up Prettier

### LINKED ISSUE #151: Add prettier config
Would make it easier to do pull requests if the prettier settings were checked into the repo, e.g. .prettierrc or in the package.json

### PR DESCRIPTION
**Pull request by illright**
I must say, I don't agree with all of the decisions that Prettier makes, but considering that it's such a ubiquitous tool, I see why we would wanna have it in our project.

Closes #151

### CODE REVIEW

**Comment by aabounegm** [comment]
Looks like we need to reconfigure eslint and stylelint to be compatible with Prettier

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,22 +41,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/) and this
...
 
-
 ## [2.1.0] - 2020-08-21
 
 First stable release with proper documentation.
 Previous, undocumented, releases can be found in [the releases section](https://github.com/illright/attractions/releases).
 
-
-[Unreleased]: https://github.com/illright/attractions/compare/v2.2.3...HEAD
+[unreleased]: https://github.com/illright/attractions/compare/v2.2.3...HEAD
```
Comment (aabounegm) [inline CHANGELOG.md:59]: Is there a way to force this to be uppercased? I would like for the heading to say "Unreleased"

**Review by aabounegm — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (illright) [reply CHANGELOG.md:59]: It shouldn't matter, should it? I thought these anchors were case-insensitive

Comment (aabounegm) [reply CHANGELOG.md:59]: Oh, ok then. I suspected that but didn't check

Comment (illright) [reply CHANGELOG.md:59]: Well, me neither

**Pull request merged**